egov-url-shortening/pom.xml

```diff
@@ -50,7 +72,11 @@
 		<dependency>
 			<groupId>org.egov.services</groupId>
 			<artifactId>tracer</artifactId>
-			<version>1.1.5-SNAPSHOT</version>
+			<version>2.0.0-SNAPSHOT</version>
```

Comment:
Any reason we added redis?

Reply:
In spring boot newer version Lettuce is used as default connector, to use Jedis connector we need to add dependency